### PR TITLE
feat(@vtmn/svelte): add `VtmnAccordion` component

### PR DESCRIPTION
**Pull request by yhafidi**
Add new component VtmnAccordion for svelte.
image

### CODE REVIEW

**Review by GaspardMathon — APPROVED** [review]

File: packages/sources/svelte/src/components/structure/VtmnAccordion/VtmnAccordion.svelte [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<script>
+  import { cn } from '../../../utils/classnames';
+
+  /**
+   * @type {boolen} Display left icon on summary.
```
Comment (GaspardMathon) [inline packages/sources/svelte/src/components/structure/VtmnAccordion/VtmnAccordion.svelte:5]:
```suggestion
   * @type {boolean} Display left icon on summary.
```

**Review by GaspardMathon — CHANGES_REQUESTED** [review]

**Comment by GaspardMathon** [comment]
I tested the accordion, nice development overall just a few things to improve : 

- It would be better if the first item of the accordion was expanded (property `open` of the `details` tag.
- In the controls, you should be able to trigger the boolean to display the icon or not.
- Same for the `disabled` property
- Same for the summary, If a enter a custom text it has to replace the current summary of the accordion

For the 3 last changes, It's ok not to have them if the Two first stories with multiple items in the accordion but for the single accordion at least.

Thanks for your contribution 💯

File: packages/showcases/svelte/stories/components/structure/VtmnAccordion/VtmnAccordion.stories.svelte [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    <VtmnButton variant="secondary" size="small">Button 2</VtmnButton>
+    <VtmnButton variant="secondary" size="small">Button 3</VtmnButton>
+  </VtmnAccordion>
+</Story>
+
+<style>
+  :global(.div-decorator) {
+    width: calc(100vw - 20%);
+  }
```
Comment (Tlahey) [inline packages/showcases/svelte/stories/components/structure/VtmnAccordion/VtmnAccordion.stories.svelte:86]: @yhafidi my bad, this one apply on all the components, we need to wrap it with another solution

**Review by GaspardMathon — DISMISSED** [review]

**Review by GaspardMathon — APPROVED** [review]

File: packages/sources/svelte/rollup.config.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ const src = {
     },
     {
       folder: 'structure',
-      components: ['VtmnDivider', 'VtmnList', 'VtmnListItem'],
+      components: ['VtmnDivider', 'VtmnList', 'VtmnListItem', 'VtmnAccordion'],
```
Comment (lauthieb) [inline packages/sources/svelte/rollup.config.js:31]:
```suggestion
      components: ['VtmnAccordion', 'VtmnDivider', 'VtmnList', 'VtmnListItem'],
```

File: packages/sources/svelte/src/components/index.js [hunk 4]
```diff
@@ -22,5 +22,6 @@ export { default as VtmnToggle } from './selection-controls/VtmnToggle/VtmnToggl
 
 // Structure
 export { default as VtmnDivider } from './structure/VtmnDivider/VtmnDivider.svelte';
+export { default as VtmnAccordion } from './structure/VtmnAccordion/VtmnAccordion.svelte';
 export { default as VtmnList } from './structure/VtmnList/VtmnList.svelte';
 export { default as VtmnListItem } from './structure/VtmnListItem/VtmnListItem.svelte';
```
Comment (lauthieb) [inline packages/sources/svelte/src/components/index.js:27]:
```suggestion
export { default as VtmnAccordion } from './structure/VtmnAccordion/VtmnAccordion.svelte';
export { default as VtmnDivider } from './structure/VtmnDivider/VtmnDivider.svelte';
export { default as VtmnList } from './structure/VtmnList/VtmnList.svelte';
export { default as VtmnListItem } from './structure/VtmnListItem/VtmnListItem.svelte';
```

File: packages/sources/svelte/src/components/structure/VtmnAccordion/VtmnAccordion.svelte [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+   */
+  export { className as class };
+  $: componentClass = cn(
+    'vtmn-accordion',
+    withIconLeft ? 'vtmn-accordion--with-icon-left' : '',
+    className,
+  );
+</script>
+
+<details class={componentClass} aria-disabled={disabled} {open}>
```
Comment (lauthieb) [inline packages/sources/svelte/src/components/structure/VtmnAccordion/VtmnAccordion.svelte:31]: I think you don't need to specify "open" prop as it's an HTML default property.
But if we want to set it, you will need to spread `$$restProps` to the `<details>` HTML block.

File: packages/showcases/svelte/stories/components/structure/VtmnAccordion/VtmnAccordion.stories.svelte [hunk 6]
```diff
@@ -0,0 +1,146 @@
+<script>
+  import { Meta, Story } from '@storybook/addon-svelte-csf';
+  import { VtmnAccordion, VtmnButton } from '@vtmn/svelte';
+  import { parameters } from '@vtmn/showcase-core/csf/components/structure/accordion.csf';
+  const argTypes = {
```
Comment (lauthieb) [inline packages/showcases/svelte/stories/components/structure/VtmnAccordion/VtmnAccordion.stories.svelte:5]: Should be moved in `@vtmn/showcase-core/csf/components/structure/accordion.csf` as we did with other files.

**Review by lauthieb — CHANGES_REQUESTED** [review]
Thanks a lot @yhafidi, just small changes :)

**Review by lauthieb — APPROVED** [review]

**Review by GaspardMathon — APPROVED** [review]

**Pull request merged**